recipes/gettext/all/conanfile.py

```diff
@@ -0,0 +1,127 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, VisualStudioBuildEnvironment, tools
+import os
+import shutil
+import glob
+
+
+class GetTextConan(ConanFile):
+    name = "gettext"
+    description = "An internationalization and localization system for multilingual programs"
+    topics = ("conan", "gettext", "intl", "libintl", "i18n")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.gnu.org/software/gettext"
+    license = "GPL-3.0-or-later"
+    settings = "os_build", "arch_build", "arch", "compiler", "build_type"
+    exports_sources = ["patches/*.patch"]
+
+    
+
+    requires = ("libiconv/1.15",
+                "libxml2/2.9.9")
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _is_msvc(self):
+        return self.settings.compiler == "Visual Studio"
+
+    @property
+    def _gettext_folder(self):
+        return "gettext-runtime"
+
+    @property
+    def _make_args(self):
+        return None
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def build_requirements(self):
+        if tools.os_info.is_windows:
+            if "CONAN_BASH_PATH" not in os.environ:
+                self.build_requires("msys2/20190524")
+        if self._is_msvc:
+            self.build_requires("automake/1.16.1")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "gettext-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def build(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)    
+        libiconv_prefix = self.deps_cpp_info["libiconv"].rootpath
+        libxml2_prefix = self.deps_cpp_info["libxml2"].rootpath
+        libiconv_prefix = tools.unix_path(libiconv_prefix) if tools.os_info.is_windows else libiconv_prefix
+        libxml2_prefix = tools.unix_path(libxml2_prefix) if tools.os_info.is_windows else libxml2_prefix
+        args = ["HELP2MAN=/bin/true",
+                "EMACS=no",
+                "--disable-nls",
+                "--disable-dependency-tracking",
+                "--enable-relocatable",
+                "--disable-c++",
+                "--disable-java",
+                "--disable-csharp",
+                "--disable-libasprintf",
+                "--disable-curses",
+                "--with-libiconv-prefix=%s" % libiconv_prefix,
+                "--with-libxml2-prefix=%s" % libxml2_prefix]
+        build = None
+        host = None
+        rc = None
+        if self.options.get_safe("shared"):
```

Comment:
This recipe has no `shared` option and because this is an installer, neither static or shared libraries should be built.
Or at least, there should none be packaged by this recipe.